advanced/index.php

```diff
@@ -70,10 +46,14 @@ function setHeader($type = "x") {
     <html lang='en'>
         <head>
             <meta charset='utf-8'>
-            $viewPort
+            <meta name='viewport' content='width=device-width, initial-scale=1'>
             <title>● $serverName</title>
-            <link rel='stylesheet' href='pihole/blockingpage.css'>
             <link rel='shortcut icon' href='admin/img/favicons/favicon.ico' type='image/x-icon'>
```

Comment:
Just throwing in the question why asset paths are relative, starting with this PR: #3263
This breaks asset access when accessing a non-existent resource inside a sub directory as `https://my.domain.org/subdir/admin/img/favicons/favicon.ico` in this particular case does not exist. Same with the logo below. Inconsistently the admin page link is still absolut?

Reply:
@PromoFaux

Reply:
https://github.com/pi-hole/pi-hole/pull/3910/commits/adaff1793cb5b64b28ac4e71a2f70a572edb77b7

Reply:
https://github.com/pi-hole/pi-hole/pull/3910/commits/adaff1793cb5b64b28ac4e71a2f70a572edb77b7